### DISCUSSION TITLE
Best way to deploy Saleor on AWS

### DISCUSSION THREAD

**Discussion by Vultik** [saleor/saleor#7987] [Deployment]
Hi, 
Thanks for this great e-commerce platform.
I'm looking for the best way to deploy Saleor on AWS, i have seen different solutions.
- AWS RDS for Postgresql db 
- EC2 instance using ECS to deploy the docker container for the backend
- S3 for the front store and the admin dashboard

I have seen also people using AWS elastic beanstalk and maybe also AWS Amplify for storefront ?
I'm curious to know how you setup Saleor on AWS :)

**Comment by karolkielecki** [comment]
@Vultik Hi! Saleor's preferred deployment method is Docker. Storefront and dashboard are static apps, so they don't require additional servers--you can use S3 for them.

**Reply by Vultik** [reply]
So to recap : 
Storefront + dashboard -> S3 with CloudFront
Postgresql DB -> Amazon RDS Postgresql 
For the backend (Django App) -> EC2 with docker (ECS) ?

For the saleor backend, I still don't know what is the best options, my need is for a small e commerce website
So do you have more advices based on your expérience to chose between a EC2 or a LightSail instance.
Furthermore, is it needed to have all in a Elastic beanstalk for a small website ?
Thanks for the support !
